cruise-control/src/yaml/endpoints/removeDisks.yaml

```
@@ -0,0 +1,69 @@
+RemoveDisksEndpoint:
+  post:
+    operationId: removeDisks
+    summary: Move all replicas from the specified disk to another disk of the same broker.
+    parameters:
+      - name: dryrun
+        in: query
+        description: Whether to dry-run the request or not.
+        schema:
+          type: boolean
+          default: true
+      - name: stop_ongoing_execution
+        in: query
+        description: Whether to stop the ongoing execution (if any) and start executing the given request.
+        schema:
+          type: boolean
+          default: false
+      - name: reason
+        in: query
+        description: Reason for request.
+        schema:
+          type: string
+        example: "Balance disk utilization across all brokers in the cluster."
+      - name: brokerid_and_logdirs
+        in: query
+        description: List of broker id and logdir pair to be demoted in the cluster.
+        schema:
+          type: object
+          additionalProperties:
+            type: array
+            items:
+              type: string
+        required: true
+        example: 101-/tmp/kafka-logs-1,101-/tmp/kafka-logs-2
```

nit:
```suggestion
        example: 101:/tmp/kafka-logs-1,101:/tmp/kafka-logs-2
```
can we use `:` as a separator between brokerid and logdir? reads more cleaner

---

I believe it is possible to do it so. However, I used the same format as for the broker id and logdir param from the demote broker endpoint, for consistency purposes.